sql-statements/sql-statement-overview.md

```diff
@@ -0,0 +1,300 @@
+
+## Resource groups
+
+<CustomContent platform="tidb">
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`ALTER RESOURCE GROUP`](/sql-statements/sql-statement-alter-resource-group.md) | Modifies a resource group. |
+| [`CALIBRATE RESOURCE`](/sql-statements/sql-statement-calibrate-resource.md) | Calibrates system resources. |
+| [`CREATE RESOURCE GROUP`](/sql-statements/sql-statement-create-resource-group.md) | Creates a new resource group. |
+| [`DROP RESOURCE GROUP`](/sql-statements/sql-statement-drop-resource-group.md) | Drops a resource group. |
+| [`QUERY WATCH`](/sql-statements/sql-statement-query-watch.md) | Manage runaway query watch list. |
+| [`SET RESOURCE GROUP`](/sql-statements/sql-statement-set-resource-group.md) | Sets a resource group. |
+| [`SHOW CREATE RESOURCE GROUP`](/sql-statements/sql-statement-show-create-resource-group.md) | Shows the CREATE statement for a resource group. |
+
+</CustomContent>
+
+<CustomContent platform="tidb-cloud">
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`ALTER RESOURCE GROUP`](/sql-statements/sql-statement-alter-resource-group.md) | Modifies a resource group. |
+| [`CREATE RESOURCE GROUP`](/sql-statements/sql-statement-create-resource-group.md) | Creates a new resource group. |
+| [`DROP RESOURCE GROUP`](/sql-statements/sql-statement-drop-resource-group.md) | Drops a resource group. |
+| [`QUERY WATCH`](/sql-statements/sql-statement-query-watch.md) | Manage runaway query watch list. |
+| [`SET RESOURCE GROUP`](/sql-statements/sql-statement-set-resource-group.md) | Sets a resource group. |
+| [`SHOW CREATE RESOURCE GROUP`](/sql-statements/sql-statement-show-create-resource-group.md) | Shows the CREATE statement for a resource group. |
+
+</CustomContent>
+
+## Utility statements
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`DESC`](/sql-statements/sql-statement-desc.md) | Describes the structure of a table. |
+| [`DESCRIBE`](/sql-statements/sql-statement-describe.md) | Describes the structure of a table. |
+| [`DO`](/sql-statements/sql-statement-do.md) | Executes an expression. |
+| [`EXPLAIN`](/sql-statements/sql-statement-explain.md) | Shows the execution plan of a query. |
+| [`TRACE`](/sql-statements/sql-statement-trace.md) | Traces query execution. |
+| [`USE`](/sql-statements/sql-statement-use.md) | Sets the current database. |
+
+## Show statements
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`SHOW BUILTINS`](/sql-statements/sql-statement-show-builtins.md) | Lists builtin functions. |
+| [`SHOW CHARACTER SET`](/sql-statements/sql-statement-show-character-set.md) | Lists character sets. |
+| [`SHOW COLLATIONS`](/sql-statements/sql-statement-show-collation.md) | Lists collations. |
+| [`SHOW ERRORS`](/sql-statements/sql-statement-show-errors.md) | Shows error messages. |
+| [`SHOW STATUS`](/sql-statements/sql-statement-show-status.md) | Shows server status. |
```

Comment:
```suggestion
| [`SHOW STATUS`](/sql-statements/sql-statement-show-status.md) | Included for compatibility with MySQL. TiDB uses Prometheus and Grafana for centralized metrics collection instead of `SHOW STATUS` for most metrics. |
```

Reply:
Maybe link to `grafana-tidb-dashboard.md` ?